### PR TITLE
#170764603: Most travelled destinations

### PR DESCRIPTION
**Pull request by KarangwaEmmy**
### Description of Task to be completed?
- info on most traveled destinations such as
- Number of requests
- Accommodation in that destination
### How should this be manually tested?
git clone this repo and checkout to this branch
npm run dev go on login a user and acquire the token
Open postman and visit http://localhost:5000/api/requests/destinations/mostTravelled

### Any background context you want to provide?
N/A

**What are the relevant pivotal tracker stories?**
#170764603
https://www.pivotaltracker.com/story/show/170764603

### Screenshots (if appropriate)
destinations

### CODE REVIEW

File: src/database/migrations/20200206110239-create-location.js [hunk 1]
```diff
@@ -9,6 +9,10 @@ export default {
     destination: {
       type: Sequelize.STRING
     },
+    visitCount: {
```
Comment (bihire) [inline src/database/migrations/20200206110239-create-location.js]: change this to "visit_count"

File: src/routes/requests.js [hunk 2]
```diff
@@ -17,6 +17,7 @@ router.post(
   AuthMiddleware.rememberMe
 );
 router.get('/search', AuthMiddleware.verifyToken, requestController.search);
+router.get('/mostTravelled', AuthMiddleware.verifyToken, requestController.MostTravelledDestination);
```
Comment (bihire) [inline src/routes/requests.js]: change this to "/destinations/mostTraveled"

File: src/controllers/requestController.js [hunk 3]
```diff
@@ -108,6 +109,10 @@ class RequestController {
         gender
       };
       const { dataValues } = await Request.create(info);
+      // const results = await sequelize.query(
```
Comment (ajimae) [inline src/controllers/requestController.js]: Remove all commented code block

**Review by ajimae — APPROVED** [review]
LGTM!

**Pull request merged**